### PR TITLE
QA project page

### PR DESCRIPTION
**Pull request by axemonkey**
## Data section

screen shot 2016-08-31 at 14 50 58
## Graphs

screen shot 2016-08-31 at 14 51 19

### CODE REVIEW

File: dashboard/assets/js/cumulative-graph.js [hunk 1]
```diff
@@ -224,7 +224,12 @@ export function plotCumulativeSpendings(project, showBurnDown, startDate, endDat
       yanchor: 'top'
     },
     shapes: shapes,
-    annotations: annotations
+    annotations: annotations,
+    margin: {
```
Comment (cliffxuan) [inline dashboard/assets/js/cumulative-graph.js:228]: nice solution. what do `t`, `l`, `r` mean exactly?

**Comment by cliffxuan** [comment]
code looks good 👍 
since we don't use `rebass` anymore, it can be removed from `package.json`

File: dashboard/assets/js/cumulative-graph.js (hunk 1 above)
Comment (axemonkey) [reply dashboard/assets/js/cumulative-graph.js:228]: `top`, `left` and `right` respectively. They default to 80px!

https://plot.ly/javascript/reference/#layout-margin

**Comment by axemonkey** [comment]
Ah yes, will do. 👍

File: dashboard/assets/js/cumulative-graph.js (hunk 1 above)
Comment (cliffxuan) [reply dashboard/assets/js/cumulative-graph.js:228]: nice

**Pull request merged**